### ISSUE TITLE
Brew maybe not upgrading to 2020.7.2 or just old reference is code.

### ISSUE THREAD

**Issue by adaptive** [cloudflare/cloudflared#222]
```bash
🍺  /usr/local/Cellar/cloudflared/2020.7.2: 3 files, 46.9MB, built in 6 seconds
Removing: /usr/local/Cellar/cloudflared/2020.7.1... (3 files, 46.9MB)
Removing: /Users/hugo/Library/Caches/Homebrew/cloudflared--2020.7.1.tgz... (20.2MB)
```
After Brew upgrade
```bash
▓▒░ cloudflared -v
cloudflared version 2020.7.1 (built 2020-07-06-1748 UTC)
```

**Comment by adaptive** [comment]
Also on Ubuntu not updating

```bash
root@apache:~# cloudflared update
root@apache:~# cloudflared -v
cloudflared version 2020.7.1 (built 2020-07-06-1751 UTC)
```

**Comment by GrahamCampbell** [comment]
You will need to restart the service after you upgrade. That won't happen automatically.

**Comment by GrahamCampbell** [comment]
I can't replicate this?

<img width="903" alt="image" src="https://user-images.githubusercontent.com/2829600/87349460-3c6b3000-c54e-11ea-9458-b6642b19327e.png">

**Comment by KommX** [comment]
In the meantime, 2020.7.3 was released and the update from 20207.1 to 2020.7.3 worked. Somehow the update to 2020.7.2 from 2020.7.1 didn't work.

**Comment by adaptive** [comment]
Seems fixed
